test/e2e/suite/certificates/additionaloutputformats.go

```diff
@@ -147,7 +161,9 @@ var _ = framework.CertManagerDescribe("Certificate AdditionalCertificateOutputFo
 
 		By("add Combined PEM to Certificate's Additional Output Formats")
 		crt.Spec.AdditionalOutputFormats = []cmapi.CertificateAdditionalOutputFormat{{Type: "CombinedPEM"}}
-		crt, err = f.CertManagerClientSet.CertmanagerV1().Certificates(f.Namespace.Name).Update(context.Background(), crt, metav1.UpdateOptions{})
+		crt.ManagedFields = nil
```

Comment:
Why set ManagedFields to `nil`? Add an explanatory comment.

Reply:
Thanks will add the comment
Managed fields cannot be included in the object applied via server-side apply https://kubernetes.io/docs/reference/using-api/server-side-apply/#apply-and-update
(I don't know exactly why, perhaps to avoid taking ownership of that field)